keyboards/tweetydabird/chameleon/readme.md

```diff
@@ -0,0 +1,28 @@
+# Chameleon 
+
+![chameleon](https://lectronz.com/lectronz/active_storage/representations/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBalFoIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--4be3beffa89240abba6874d8ef242fc7b0b10994/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdCem9MWm05eWJXRjBTU0lKU2xCRlJ3WTZCa1ZVT2hSeVpYTnBlbVZmZEc5ZmJHbHRhWFJiQjJrQ3dBTnBBb0FDIiwiZXhwIjpudWxsLCJwdXIiOiJ2YXJpYXRpb24ifX0=--3e445397da8bf51b531321877b7934091a51a638/IMG_1311.JPEG)
```

Comment:
could this image be re-hosted on Imgur?

Reply:
Sure, not sure why though? It should be pretty stable where it is, since that's where I sell the PCB, but I'll happily put it on Imgur as well.